.byebug_history

```diff
@@ -0,0 +1,17 @@
+quit 
```

Comment:
I imagine we should probably leave this file out of the merge. If you don't mind, would you remove this from the PR and pop in `.byebug_history` to the `.gitignore` file?